README.md

```diff
@@ -22,8 +29,83 @@ then open browser [here](http://localhost:1234/index.html?p=12345)
 
 You should now see something like the following 👍
 
-![](./doc/2020-11-20-18-45-33.png)
+![](./doc/media/2020-11-20-18-45-33.png)
+
+<!-- 
+
+plantuml -tsvg README.md 
+
+-->
+
+## Sequence diagram : _WISP light_
+
+[Here](https://io-p-cdnendpoint-iopay.azureedge.net/index.html?p=12112) the public endpoint to `io-pay` (alias _Wisp Light_)
+
+<!-- 
+@startuml doc/uml_diag/seqdiag-wisplight
+
+autonumber 
+participant web as "WEB/User"
+participant wl as "IO PAY"
+participant api as "pp-rest"
+participant logging as "pp-logging"
+
+
+autonumber 
+participant web as "WEB/User"
+participant wl as "WISP"
+participant api as "pp-rest"
+participant logging as "pp-logging"
+
+web -> wl : redirect to <IO_PAY_URL>/enterEmail.html?p<ID_PAYMENT>
+
+wl -> api : **GET** payments/{idPayment}/actions/check
+api -> wl : Payment
+note over wl: `onload` page to check the correctness of the payment and retrieve the associated data \n This call shall be unique to the entire payment session because it isn't idempotent \n Although if it's a GET it writes on PM's DB and checks if payment already exists.
+
+web -> web :The user insert/change mail and click `Continua`
+
+web -> wl :The user insert credit card info and accepts terms & privacy policy then click `Continua`
+
+wl -> api : **POST** users/actions/start-session{email, idPayment}
+api -> wl : OK {sessionToken,sessionUser}
+
+wl -> api : **POST** /users/actions/approve-terms{terms, privacy}
+api -> wl : OK
+note over wl: Approve Term and Condition
+wl -> logging : **PUT** db-logging/logging/db/save event
+
+api -> wl : OK
+
+wl -> api : **POST** /wallet(wallet) 
+api -> wl : OK
+note over wl: addWallet returns a response with payment data\n (wallet, data, card, and psp which contains the fixed costs amount
+
+wl -> wl : Summary page (wallet/summary)
+
+web -> wl :Confirm your payment, click on `Paga`
+
+wl -> api : **POST** /payments/{idPayment}/actions/pay?language?it {idWallet,idPayment}
+
+
+api -> wl : OK (datiPagamento)
+
+wl -> logging : **PUT** db-logging/logging/db/save event
+
+wl -> web : UrlCheckout3ds 
```

Comment:
_REMINDER_ : @infantesimone i think that 3DS scenario should be update after WISP/PM code-review and 3DS2